packages/components/releaseNotes/components.md

```diff
@@ -1,6 +1,10 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version 2.202.7
+*Released*: 9 August 2022
+* Pull forward fixes from release22.7-SNAPSHOT
```

Comment:
maybe update comment to "Merge release22.7-SNAPSHOT to release22.8-SNAPSHOT" since it's not going to develop directly

Reply:
done.